tasks/main.yml

```diff
@@ -0,0 +1,169 @@
+---
+- name: Import dovecot key
+  apt_key:
+    url: https://repo.dovecot.org/DOVECOT-REPO-GPG
+
+- name: Add dovecot repository
+  apt_repository:
+    repo: deb https://repo.dovecot.org/ce-{{ dovecot_release }}-latest/debian/{{ ansible_distribution_release }} {{ ansible_distribution_release }} main
+    filename: dovecot
+
+- name: Install dovecot dependencies
+  apt:
+    name:
+      - dovecot-core
+      - dovecot-imapd
+      - dovecot-ldap
+      - dovecot-lucene
+      - dovecot-managesieved
+      - dovecot-sieve
+
+- name: Create configuration file
+  template:
+    src: dovecot.conf.j2
+    dest: /etc/dovecot/dovecot.conf
+    owner: root
+    group: root
+    mode: 0640
+  notify: Reload dovecot
+
+- name: Create LDAP passdb configuration
+  template:
+    src: ldap.conf.j2
+    dest: /etc/dovecot/ldap-passdb.conf
+    owner: root
+    group: root
+    mode: 0640
+  notify: Reload dovecot
+
+# I could use the same file, but the documentation
+# suggests different filenames to ensure different
+# connections are used which boosts performance as
+# the userdb lookups can be made async.
+- name: Create LDAP userdb configuration
+  file:
+    src: ldap-passdb.conf
```

Comment:
I think the file is missing or should the user place it in `files/` ?

Reply:
It's missing because we are only creating a symlink